lib/locales/es.ts

```diff
@@ -477,6 +477,15 @@ export const locale: Locale = {
     odile_of_alsace_abbess_patroness_of_alsace: 'Santa Odilia, abadesa, Patrona de Alsacia',
     olga_of_kiev: 'Santa Olga',
     our_lady_help_of_christians: 'Nuestra Señora, Auxilio de los Cristianos',
+    /*
+     * src: https://gcatholic.org/calendar/2024/CR-en.htm
+     */
+    our_lady_of_angels: 'Nuestra Señora de los Ángeles',
```

Comment:
This should be removed, as it is not used in any calendar in romcal.

```suggestion
```

Reply:
but it's the main l10n key, so i believe this would cause issues

Reply:
imo we should remove the necessity of custom l10n keys for title amendments and those titles should be localized, rather than doubling those translations